MaxText/decode.py

```diff
@@ -232,11 +237,15 @@ def decode_loop(config, state=None):
 
   new_text, _ = decode_tokens([int(x[0,0]) for x in outputs], sp_tokenizer)
   max_logging.log(f"Completion: `{config.prompt}` -> `{new_text}`")
+  if config.prompt == "I love to" and config.model_name == "llama2-7b":
```

Comment:
I don't understand this? Shouldn't it be

```
if config.autoregressive_decode_assert != "":
  assert new_text==config.autoregressive_decode_assert, f"generated text mismatch {new_text=} {config.autoregressive_decode_assert=}"
```

Reply:
Done